edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.Test/KubernetesRuntimeInfoProviderTest.cs

```diff
@@ -185,21 +185,6 @@ public void ConstructorChecksNull()
             Assert.Throws<ArgumentNullException>(() => new KubernetesRuntimeInfoProvider("namespace", client.Object, null));
         }
 
-        [Fact]
-        public async void GetModuleLogsTest()
```

Comment:
Why are you removing it?

Reply:
I removed the functionality from edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubernetesRuntimeInfoProvider.cs. It was never fully implemented for k8s, and after talking to david we decided to remove it. The feature isn't very useful on k8s, which has its own ways to get logs.